engines/sword2/detection_tables.h

```diff
@@ -110,7 +95,7 @@ static const Sword2GameDescription gameDescriptions[] = {
 	{
 		{
 			"sword2",
-			"EU",
+			"Speech",
 			AD_ENTRY3s("general.clu", "31db8564f9187538f24d9fda0677f666", 7059728,
```

Comment:
Why was the "EU" flag removed? This indicates a different release (EU vs. US)

Reply:
Recovered